api4/system.go

```diff
@@ -262,14 +262,16 @@ func postLog(c *Context, w http.ResponseWriter, r *http.Request) {
 		msg = msg[0:399]
 	}
 
+	msg = "Logs API Endpoint Message: " + msg
```

Comment:
Is it necessary to prefex it if we have the structured type? If we feel it's nessisary maybe `Client Logs API Endpoint Message: ` to be more clear.

Reply:
0/5 - It makes it even more clear that this is a client message, but I don't see it as a hard requirement. Since we don't document the structured fields itself it may be good to make sure admins and other staff understand that this is a client-side controlled message.@wiersgallak @hmhealey thoughts?

Reply:
I could see different text here, but I agree that it makes it clearer if we have this in the log message itself. The text feels like the most important information in the log message, and the other fields just seem like extra information that may get skipped over

Reply:
Alright. We can keep it then. But let's add client in there to make it more clear where it's coming from.

Reply:
I am in favor of making this text more explicit and obvious.

Reply:
@crspeller Adjusted the wording as suggested.